scholly/base/src/layouts/layout1/components/landingPage/LandingPage.js

```diff
@@ -1,14 +1,29 @@
 /**
  * This file output landing page (homepage) of client-site.
  */
-import React from 'react';
-import { Container } from 'react-bootstrap';
+import React, { useState, useEffect, useRef } from 'react';
+import {
+  Row, Col, Accordion, Container,
+} from 'react-bootstrap';
 import { Helmet } from 'react-helmet';
-import { TEAM_HOMEPAGE, TEAM_INFO } from '../../../../global/data';
+import { Link } from 'react-router-dom';
+import {
+  TEAM_PUBLICATIONS, TEAM_HOMEPAGE, TEAM_INFO, TEAM_SITE_METADATA,
+} from '../../../../global/data';
+import TwitterFeed from '../twitter/TwitterFeed';
+import Publication from '../publications/publication/Publication';
 
 const landingPage = () => {
   const homepageData = TEAM_HOMEPAGE;
-  const { teamName } = TEAM_INFO;
+  const themeOption = TEAM_SITE_METADATA.template.theme;
+  const { teamName, twitterHandle } = TEAM_INFO;
+  const pubs = TEAM_PUBLICATIONS;
+  const [height, setHeight] = useState(0);
+  const ref = useRef(null);
+
+  useEffect(() => {
+    setHeight((ref.current.clientHeight > 600) ? ref.current.clientHeight : 600);
```

Comment:
condense this logic by using a min function to ensure we're having 600 as the min

Reply:
I think we should use the max function to ensure that 600 remains the minimum height for the twitter column.

Reply:
sgtm